en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -223,3 +191,63 @@ aerial video optimal value may be much smaller to get smooth response.)
+The PID-Gains should be chosen such that tracking is as tight as possible. Before doing any position/velocity control related tuning,
+turn off all [higher-level position controller tuning gains](advanced_mc_position_tuning.md).
+
+- [MPC_ACC_HOR_MAX](parameter_reference.md#MPC_ACC_HOR_MAX): 1000
+- [MPC_ACC_HOR](parameter_reference.md#MPC_ACC_HOR) : 1000
+- [MPC_DEC_HOR_SLOW](parameter_reference.md#MPC_DEC_HOR_SLOW) : 1000 
+- [MPC_ACC_UP_MAX](parameter_reference.md#MPC_ACC_UP_MAX) : 1000 
+- [MPC_ACC_DOWN_MAX](parameter_reference.md#MPC_ACC_DOWN_MAX) : 1000 
+- [MPC_JERK_MAX](parameter_reference.md#MPC_JERK_MAX) : 0 
+- [MPC_JERK_MIN](parameter_reference.md#MPC_JERK_MIN) : 1
+ -->
+
+
+### Airmode & Mixer Saturation
+
+The rate controller outputs roll, pitch, yaw and thrust commands, which need to be converted into individual motor thrust commands. 
+This step is called mixing.
+
+It can happen that one of the motor commands becomes negative, for example for a low thrust and large roll command (and similarly it can go above 100%). 
+This is a mixer saturation. 
+It is physically impossible for the vehicle to execute these commands (except for reversible motors). PX4 has two modes to resolve this:
+- Either by reducing the commanded roll such that none of the motor commands is
+  below zero (Airmode disabled). In the extreme case where the commanded thrust
+  is zero, it means that no attitude correction is possible anymore, which is
+  why a minimum thrust is always required for this mode.
+- Or by increasing (boosting) the commanded thrust, such that none of the motor
+  commands is negative (Airmode enabled). This has the big advantage that the
+  attitude/rates can be tracked correctly even at low or zero throttle. It
+  generally improves the flight performance.
+
+  However it increases the total thrust which can lead to situations where the
+  vehicle continues to ascend even though the throttle is reduced to zero. For a
+  well-tuned, correctly functioning vehicle it is not the case, but for example
+  it can happen when the vehicle strongly oscillates due to too high P tuning
+  gains.
+
+Both modes are shown below with a 2D illustration for two motors and a roll
+command <span style="color:#9673A6">r</span>. On the left motor
+<span style="color:#9673A6">r</span> is added to the commanded thrust, while on
+the right motor it is substracted from it.
+The motor thrusts are in <span style="color:#6A9153">green</span>.
+With Airmode enabled, the commanded thrust is increased by
+<span style="color:#B85450">b</span>. When it is disabled,
+<span style="color:#9673A6">r</span> is reduced.
+
+  ![Airmode](../../images/mc_pid_tuning/MC_PID_tuning-Airmode.svg)
+<!-- The drawing is on draw.io: https://drive.google.com/file/d/1N0qjbiJX6JuEk2I1-xFvigLEPKJRIjBP/view?usp=sharing
+     On the first Tab
+-->
+
+If mixing becomes saturated towards the upper bound, the Airmode logic is always used (the commanded thrust is reduced).

```

Comment:
@bkueng In airmode isn't the commanded thrust increased to meet a required roll? So for upper bound I would expect to **not** use airmode so that we don't saturate.

Reply:
It depends on the explanation. The way I meant it is that with airmode, the mixer is allowed to change the commanded thrust. This can be towards positive or negative direction. Which is why the logic for the upper bound is the same as for airmode, just with different signs.

Reply:
@bkueng What I think you are saying is that whether airmode is turned on or off, if the total commanded thrust for any motor is higher than 100% the commanded thrust will be reduced until no motor is being requested to deliver more than 100%. 

This is confusing because 
- we only talk in airmode about increasing the commanded thrust, and we only talk about decreasing the commanded thrust when not in airmode
- This actually sounds more like the "not airmode" type of behaviour.

Perhaps change to 

> If mixing becomes saturated towards the upper bound the commanded thrust is reduced to ensure that no motor is commanded to deliver more than 100% thrust. This behaviour is true irrespective of whether Airmode is enabled or disabled.

A diagram would also be nice for this case.

Reply:
> and we only talk about decreasing the commanded thrust when not in airmode

This is not correct, we decrease commanded roll when not in airmode, not commanded thrust. This is why the logic is the same as in Airmode: it changes the commanded thrust, rather than changing the commanded roll.
If it's too confusing, we can add a diagram. But it will look like the Airmode case.

The changed text is good, but I'd like to mention that it's like the Airmode logic.